app/assets/javascripts/admin/analytics.js

```diff
@@ -1,4 +1,4 @@
-(function (GOVUK) {
+;(function (GOVUK) {
```

Comment:
This looks weird. Wonder what rule is causing that?

Reply:
Ah, thanks for finding that!